Gemfile

```diff
@@ -38,12 +38,16 @@ gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'devise'
 gem 'will_paginate', '~> 3.1.0'
 gem 'rails-html-sanitizer'
-gem 'letsencrypt_plugin'
+#gem 'letsencrypt_plugin'
```

Comment:
I commented the gem 'letsencrypt_plugin' because heroku already provides encryption for you heroku domain..I had some problems with lets encrypt with my heroku server